doc/application/application.rst

```diff
@@ -134,19 +483,19 @@ Overview
 ========
 
 CMake is used to build Zephyr. A CMake build is done in two
-stages. The first stage is called Configuration, during Configuration
-the CMakeLists.txt build scripts are executed. At the end of
-Configuration, CMake will have an internal model of the Zephyr build
-and can generate build scripts that are native to the host platform.
-
-CMake supports generating several build systems, but only Ninja and
-Make are tested and supported by Zephyr. After Configuration and
-Generation the user can begin the Build stage by executing the
-generated build system. The build system will for most code changes be
-able to re-build the application without involving CMake. But for
-certain configuration changes the Configuration step will need to be
-executed again, either automatically by the build system, or even
-manually in some cases.
+stages. The first stage is called **configuration**. During configuration,
+the CMakeLists.txt build scripts are executed. After configuration is finished,
+CMake has an internal model of the Zephyr build, and can generate build scripts
+that are native to the host platform.
+
+CMake supports generating scripts for several build systems, but only Ninja and
+Make are tested and supported by Zephyr. After configuration, the user can
```

Comment:
Previously you used the more active "you" (thank you for doing that) but now you're using the passive "the user".  It's best to be consistent, and use "you" and keep it active.  Also, we're missing the words "second stage" to go along with your intro about there being two stages.  Let's change this to:

     After configuration, you begin the second or **build** stage by executing the ...

Reply:
That makes good sense, will do.